### PR TITLE
add deterministic to ci build

### PR DESCRIPTION
**Pull request by jennyf19**
Fixes # .
#2754 

image

image

### CODE REVIEW

**Review by jmprieur — APPROVED** [review]
LGTM
Thanks @jennyf19

**Comment by trwalke** [comment]
Seems like there is some strange error on the mac build
https://identitydivision.visualstudio.com/IDDP/_build/results?buildId=798721&view=logs&j=d321d9b0-c4ab-5ab5-b698-5efd5117e208&t=5f1f9caf-45b3-5832-6b9e-f1d7a51bb48a&l=4698

**Review by trwalke — APPROVED** [review]
LGTM. One minor build issue on mac. 
Thanks @jennyf19

**Comment by jmprieur** [comment]
@trwalke is the build issue on system center due to this change? or was it pre-existing?

**Comment by trwalke** [comment]
@jmprieur  Seems like it is coming from this PR. Current master does not have this issue
https://identitydivision.visualstudio.com/IDDP/_build/results?buildId=798776&view=logs&j=d321d9b0-c4ab-5ab5-b698-5efd5117e208

Also, the error says 

"error : SourceRoot items must include at least one top-level (not nested) item when DeterministicSourcePaths is true"

so maybe this is triggering something called DeterministicSourcePaths on the mac build?

**Comment by bgavrilMS** [comment]
@jennyf19 - i (hopefully) fixed the CI build on Mac. Not sure why it behaves differently on Windows. It looks we have to tell msbuild where the root directory of the repo is.

**Review by jmprieur — APPROVED** [review]
LGTM
Thanks @jennyf19 and @bgavrilMS

File: src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj [hunk 1]
```diff
@@ -49,19 +49,21 @@
     <!-- Build symbol package (.snupkg) to distribute the PDB containing Source Link -->
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-    <EmbedUntrackedSources>true</EmbedUntrackedSources>
```
Comment (bgavrilMS) [inline src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj:52]: it's already there at line 48...

**Review by bgavrilMS — APPROVED** [review]

**Pull request merged**